tests/v1/build.bats

```diff
@@ -1,19 +1,22 @@
 #!/usr/bin/env bats
 
 load "${BATS_PLUGIN_PATH}/load.bash"
-load '../../lib/shared'
+load '../../lib/shared.bash'
 
- export DOCKER_STUB_DEBUG=/dev/stdout
- export BUILDKITE_AGENT_STUB_DEBUG=/dev/stdout
+# export DOCKER_COMPOSE_STUB_DEBUG=/dev/stdout
 # export BATS_MOCK_TMPDIR=$PWD
 
 setup_file() {
-  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_CLI_VERSION=2
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_CLI_VERSION=1
 }
 
 teardown() {
-  # plugin leaves override files around
-  rm -f docker-compose.buildkite*-override.yml
+  # some test failures may leave this file around
+  rm -f docker-compose.buildkite-1-override.yml
```

Comment:
I'm sure this is change is reverted and corrected in another PR, but highlighting it

Reply:
Good catch... applied it again because I had to resolve a few conflicts anyways